Makefile

```
@@ -53,15 +53,17 @@ LDFLAGS := \
 .PHONY: $(APM_SERVER_FIPS_BINARIES)
 $(APM_SERVER_FIPS_BINARIES):
 	docker run --privileged --rm "tonistiigi/binfmt:latest@sha256:1b804311fe87047a4c96d38b4b3ef6f62fca8cd125265917a9e3dc3c996c39e6" --install arm64,amd64
-	# rely on Dockerfile.fips to use the go fips toolchain
-	docker buildx build --platform "$(GOOS)/$(GOARCH)" --build-arg GOLANG_VERSION="$(shell go list -m -f '{{.Version}}' go)" -f ./packaging/docker/Dockerfile.fips -t apm-server-fips-image-temp .
 	# remove any leftover container from a failed task
-	docker rm apm-server-fips-cont || true
-	docker create --name apm-server-fips-cont apm-server-fips-image-temp
+	docker container rm apm-server-fips-cont || true
+	docker image rm apm-server-fips-image-temp || true
```

Do you think we can create a new make goal and use something like?

```suggestion
clean-docker-%:
	docker container rm $@-cont || true
	docker image rm $@-image-temp || true
...

	$(MAKE) clean-docker-apm-server-fips
	...
	$(MAKE) clean-docker-apm-server-fips
```